sdk/OoyalaSkinSDK/discoveryPanel.js

```diff
@@ -69,6 +81,30 @@ var DiscoveryPanel = React.createClass({
     }
   },
 
+  onStatusPressed: function() {
+    this.setState({showCircularStatus: false});
+  },
+
+  setCounterTime: function(time) {
+    this.setState({
+      currentCounterVal: time,
+      counterLimit: time,
+      showCircularStatus: true,
+    });
+  },
+
+  updateTimer: function(row) {
+    if (this.state.currentCounterVal == 0) {
+      this.onRowSelected(row);
+    } else {
+      var self = this;
+      setTimeout(function() {
```

Comment:
i wouldn't say dump it yet.  I think it's okay, and if you have any new interesting ideas, let me know before coding it up.  it's more of a curious case to think about